<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>38</width>
    <height>38</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Operation">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <addaction name="action_Initlize"/>
    <addaction name="action_Run"/>
    <addaction name="action_Stop"/>
    <addaction name="action_Clean"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionH_elp"/>
    <addaction name="action_About"/>
    <addaction name="actiontest"/>
   </widget>
   <widget class="QMenu" name="menu_parallel">
    <property name="title">
     <string> &amp;Parallel</string>
    </property>
    <addaction name="action_Decompose"/>
    <addaction name="action_Reconstruct"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_parallel"/>
   <addaction name="menu_Operation"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>38</width>
     <height>38</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_Decompose"/>
   <addaction name="action_Reconstruct"/>
   <addaction name="separator"/>
   <addaction name="action_Initlize"/>
   <addaction name="action_Run"/>
   <addaction name="action_Stop"/>
   <addaction name="separator"/>
   <addaction name="action_Clean"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="actionH_elp">
   <property name="text">
    <string>H&amp;elp...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_ReStart">
   <property name="text">
    <string>&amp;Restart</string>
   </property>
  </action>
  <action name="action_Clean">
   <property name="text">
    <string>&amp;Clean</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="action_Decompose">
   <property name="text">
    <string>&amp;Decompose</string>
   </property>
  </action>
  <action name="action_Reconstruct">
   <property name="text">
    <string>&amp;Reconstruct</string>
   </property>
  </action>
  <action name="action_Initlize">
   <property name="text">
    <string>&amp;Initlize</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
